db/house-plants.json

```diff
@@ -0,0 +1,3514 @@
+[
+  {
+    "id": 0,
+    "latin": "Aeschynanthus lobianus",
+    "family": "Gesneriaceae",
+    "common": ["Lipstick"],
+    "category": "Hanging",
+    "origin": "Java",
+    "climate": "Tropical",
+    "tempmax": { "celsius": 32, "fahrenheit": 89.6 },
+    "tempmin": { "celsius": 14, "fahrenheit": 57.2 },
```

Comment:
Some of these columns are not present in the Plant model, are we going to add more columns?

Reply:
No we don't need the extra data. Just the ones in the model for now.